src/main/java/org/gridsuite/explore/server/RestResponseEntityExceptionHandler.java

```diff
@@ -56,6 +58,8 @@ protected ResponseEntity<Object> handleAllException(Exception exception) {
         }
         if (exception instanceof HttpStatusCodeException) {
             return ResponseEntity.status(((HttpStatusCodeException) exception).getStatusCode()).body(exception.getMessage());
+        } else if (exception instanceof HandlerMethodValidationException || exception instanceof MethodArgumentNotValidException) {
```

Comment:
There is no argument annotated with Valid in the controller, so the MethodArgumentNotValidException is not usefull, no ?

Reply:
Yes, i removed the validation so i should remove this too